pkg/compute/init.go

```diff
@@ -4,33 +4,25 @@ import (
 	"crypto/rand"
 	"fmt"
 	"io"
-	mathRand "math/rand"
-	"net"
 	"os"
 	"path/filepath"
 	"regexp"
 	"strconv"
 	"strings"
 	"time"
 
-	petname "github.com/dustinkirkland/golang-petname"
 	"github.com/fastly/cli/pkg/api"
 	"github.com/fastly/cli/pkg/common"
 	"github.com/fastly/cli/pkg/compute/manifest"
 	"github.com/fastly/cli/pkg/config"
 	"github.com/fastly/cli/pkg/errors"
 	"github.com/fastly/cli/pkg/filesystem"
 	"github.com/fastly/cli/pkg/text"
-	"github.com/fastly/go-fastly/v3/fastly"
+
 	"gopkg.in/src-d/go-git.v4"
 	"gopkg.in/src-d/go-git.v4/plumbing"
 )
 
-const (
```

Comment:
10. The changes in `init` are actually much simpler as we're primarily deleting code, and the only additional thing I have done is refactored the remaining logic into separate functions.